torch/utils/tensorboard/__init__.py

```diff
@@ -1,12 +1,12 @@
 import tensorboard
-from distutils.version import LooseVersion
+from packaging import version  # type: ignore[import]
```

Comment:
still think the ignore solves the mypy issue but should not.

Reply:
I don't know why I have looked into, but there are actually **3** `mypy.ini` configs. I understand why we use `mypy.ini` and `mypy-strict.ini`. But the uncommented `mypy-nofollow.ini`? From the commit message I got a hint. 

Which of them is used in this pipeline? I don't know but it should be the normal `mypy.ini` or `mypy-nofollow.ini` but not the strict one.